adhocracy4/polls/static/PollDetail/PollQuestions.jsx

```diff
@@ -314,31 +325,32 @@ class PollQuestions extends React.Component {
         <>{this.state.showResults
           ? (
             <div className="pollquestionlist-container">
-              {this.state.result.map((q, idx) => (
+              {this.state.result.map((question, idx) => (
                 <PollResults
                   key={idx}
-                  question={q}
+                  question={question}
                 />
               ))}
+              <Alert onClick={() => this.removeAlert()} {...this.state.alert} />
```

Comment:
so success alert is shown on results page as discussed with Marcelo, else it wouldn't be seen